kern/vmx.c

```diff
@@ -1782,7 +1781,7 @@ static void vmx_handle_external_interrupt(struct vmx_vcpu *vcpu, u32 exit_intr_i
 		register unsigned long current_stack_pointer asm(_ASM_SP);
 		vector =  exit_intr_info & INTR_INFO_VECTOR_MASK;
 		desc = (gate_desc *)vcpu->idt_base + vector;
-		entry = gate_offset(*desc);
+		entry = gate_offset(desc);
```

Comment:
`gate_offset` now accepts a pointer rather than a value:

https://github.com/torvalds/linux/commit/64b163fab684e3de47aa8db6cc08ae7d2e194373#diff-35bcd00365a749ba6cfa246a7dc86a68R96